MovieQuiz/Services/ResultAlertPresenter.swift

```diff
@@ -20,8 +20,11 @@ class ResultAlertPresenter {
         self.controller = controller
         self.actionHandler = actionHandler
     }
-    func show() {
+    func show(_ id: String? = nil) {
         let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        if id != nil {
```

Comment:
Так код будет почище, и переменная сразу не опциональная :) 
```suggestion
        if let id = id {
```